modules/openy_gc_auth/src/GCUserAuthorizer.php

```diff
@@ -72,6 +72,14 @@ public function authorizeUser($name, $email, array $extra_data = []) {
         $account->save();
       }
     }
+    // Redirecting user login page.
+    $userRolesArray = ['administrator', 'site_owner'];
```

Comment:
@kairamkondarajesh as I mentioned this morning, `site_owner` is a Y Cloud specific role and should not be hard-coded here, but `virtual_ymca_editor` should be here.